3rd PPP (#98)
itchysats/10101: docs/ppp/iteration_3_01112022.md

### Plans 📆

- Build a Lightning Full node using LDK and Flutter and open a channel automatically [https://github.com/itchysats/10101/issues/10](https://github.com/itchysats/10101/issues/10).

> - Build out frontend based on the [wireframes](https://www.figma.com/file/zUBJpQ0v8cx6DHivS10OeI/TenTenOneWireFrame).

Reviewer: This is the main focus for next view.

```suggestion
- Build the trade view based on the [wireframes](https://www.figma.com/file/zUBJpQ0v8cx6DHivS10OeI/TenTenOneWireFrame).
```

> - Build the trade view based on the [wireframes](https://www.figma.com/file/zUBJpQ0v8cx6DHivS10OeI/TenTenOneWireFrame).

Reviewer 2: ```suggestion
- Build the trade view based on the [wireframes](https://www.figma.com/file/zUBJpQ0v8cx6DHivS10OeI/TenTenOneWireFrame) as part of a Flutter app. Flutter allows to us to ship us to multiple devices out of the box.
```

> - Add a custom output to a Lightning Channel. This will be the base for adding DLCs to a channel. [https://github.com/itchysats/10101/issues/54](https://github.com/itchysats/10101/issues/54)

Reviewer: @luckysori are we going to publish the blog post? https://github.com/itchysats/10101/issues/14

Reviewer 3: As discussed during our demo call, let's publish it during the next iteration 😉

Reviewer 3: We can include it in this list then.

### Progress ✅

- Added an on-chain wallet powered by [BDK](https://github.com/bitcoindevkit/bdk) to our app. We generate a wallet basde on a Bip39 seed phrase [https://github.com/itchysats/10101/issues/68](https://github.com/itchysats/10101/issues/68).

> - Show balance in the wallet. [https://github.com/itchysats/10101/issues/90](https://github.com/itchysats/10101/issues/90).

Reviewer 2: Here is a screenshot if you want to add it but it does not look like much yet:

![image](https://user-images.githubusercontent.com/5557790/199188814-c1d879cc-acbe-412b-8bd2-a0efa6b29af1.png)

Once the navigation bars and FABs (floating action buttons) are in it will look more like an app :)

> - Show balance in the wallet. [https://github.com/itchysats/10101/issues/90](https://github.com/itchysats/10101/issues/90).

Reviewer 2: ```suggestion
- Set up Flutter and `flutter-rust-bridge` so we can stick to rust for the beefy parts but have a beautiful cross-platform app based on Flutter.
- Show balance in the wallet screen. [https://github.com/itchysats/10101/issues/90](https://github.com/itchysats/10101/issues/90).
- Work on the Wallet backup screens. [https://github.com/itchysats/10101/pull/81](https://github.com/itchysats/10101/pull/81)
- Work on navigation between Wallet and CFD trading page. [https://github.com/itchysats/10101/issues/90](https://github.com/itchysats/10101/issues/96).
```

> - Made progress on our fork of [rust-lightning](https://github.com/itchysats/rust-lightning). While we do not have a full end-to-end test yet we feel confident that it's just a few lines of code away :)

Reviewer 3: I suppose we can now say that we've added a custom output to the commitment transaction!
